tests/BinaryOperationTest.php

```diff
@@ -790,32 +848,23 @@ function example(object $foo): string
     }
 
     /**
-     * @return iterable<string,array{string,error_message:string,1?:string[],2?:bool,3?:string}>
+     * @return iterable<string,array{code:string,error_message:string,ignored_issues?:array<string>,php_version?:string}>
      */
     public function providerInvalidCodeParse(): iterable
     {
         return [
             'badAddition' => [
-                '<?php
+                'code' => '<?php
                     $a = "b" + 5;',
                 'error_message' => 'InvalidOperand',
             ],
-            'differingNumericTypesAdditionInStrictMode' => [
-                '<?php
-                    $a = 5 + 4.1;',
-                'error_message' => 'InvalidOperand',
-                'error_levels' => [],
-                'strict_mode' => true,
-            ],
-            'concatenationWithNumberInStrictMode' => [
```

Comment:
This test was dropped?

Reply:
No, it was moved up into a custom section to remove the `strict_mode` argument from the trait signature